### PR TITLE
Add jwt grant guide

### PR DESCRIPTION
**Pull request by sherenewso2**

### CODE REVIEW

File: en/docs/fragments/pickup-dispatch-saml.md [hunk 1]
```diff
@@ -10,8 +10,7 @@
 
 4. Click **Register**.
     
-5. Expand the **Inbound Authentication Configuration** section and then the **OAuth/OpenID Connect Configuration** and click
-    **Configure.**   
+5. Expand the **Inbound Authentication Configuration** section and then the **SAML2 Web SSO Configuration** and click **Configure.**
```
Comment (gomathyk) [inline en/docs/fragments/pickup-dispatch-saml.md:13]: then the **SAML2 Web SSO Configuration** section. Click **Configure**.

File: en/docs/fragments/pickup-manager-saml.md [hunk 2]
```diff
@@ -6,8 +6,7 @@
 
 2. Click **Service Providers >Add** and enter "pickup-dispatch" as the **Service Provider Name** and click **Register**.
```
Comment (gomathyk) [inline en/docs/fragments/pickup-manager-saml.md:7]: A space after > ?

Shall we write "click Register" as a separate sentence in the same step?

File: en/docs/fragments/pickup-manager-saml.md [hunk 3]
```diff
@@ -6,8 +6,7 @@
...
     
-3. Expand the **Inbound Authentication Configuration** section and then the **OAuth/OpenID Connect Configuration** and click
-    **Configure.**   
+3. Expand the **Inbound Authentication Configuration** section and then the **SAML2 Web SSO Configuration** and click **Configure.**
```
Comment (gomathyk) [inline en/docs/fragments/pickup-manager-saml.md:9]: the **SAML2 Web SSO Configuration** section. Click **Configure**.

File: en/docs/fragments/travelocity.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+
+{!fragments/deploying-sample-apps.md!}
+
+### Register a service provider
+
+1. Log in to the [Management Console](insertlink) using admin/admin credentials. 
+
+2. Click **Service Providers >Add**.
```
Comment (gomathyk) [inline en/docs/fragments/travelocity.md:8]: A space after >

File: en/docs/fragments/travelocity.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+3. Enter "travelocity.com" as the **Service Provider Name**.
+
+4. Click **Register**.
+    
+5. Expand the **Inbound Authentication Configuration** section and then the **SAML2 Web SSO Configuration**.
```
Comment (gomathyk) [inline en/docs/fragments/travelocity.md:14]: remove the before **SAML2 Web SSO Configuration** or add "section" after it.

File: en/docs/fragments/travelocity.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+6. Click **Configure.**   
+
+7. Enter the following configurations:
+    1.  **Issuer** : travelocity.com
+
+    2.  **Assertion Consumer URL** : http://wso2is.local:8080/travelocity.com/home.jsp                    
+        Click **Yes**, in the message that appears.
+
+        !!! tip
+            For more information on these fields and other advanced configurations
```
Comment (gomathyk) [inline en/docs/fragments/travelocity.md:25]: configurations, refer (remove comma)

File: en/docs/fragments/travelocity.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+            refer, [Advanced SAML Configurations](../../guides/authentication/saml-app-config-advanced)
+
+8. Select the following checkboxes:
+    - **Enable Attribute Profile**
+    - **Include Attributes in the Response Always**
+
+9. Click **Register** to save the changes. Now you are sent back to the Service Providers page.
```
Comment (gomathyk) [inline en/docs/fragments/travelocity.md:32]: Shall we highlight Service Providers?

File: en/docs/guides/access-delegation/configure-jwt-grant.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+    For more information about these configurations, see [Identity Provider Configurations](insertlink).
+
+5. Click **Register**. 
+
+----
+
+{!fragments/register-a-service-provider.md!}
+
+3. Expand **Inbound Authentication Configuration** section and then **OAuth/OpenID Connect Configuration**.
```
Comment (gomathyk) [inline en/docs/guides/access-delegation/configure-jwt-grant.md:24]: Shall we remove "section"?

File: en/docs/guides/access-delegation/configure-jwt-grant.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+	``` 
+
+	``` powershell tab="Sample Request"
+	curl -i -X POST -H 'Content-Type: application/x-www-form-urlencoded' -u bBhEoE2wIpU1zB8HA3GfvZz8xxAa:RKgXUC3pTRQg9xPpNwyuTPGtnSQa -k -d 'grant_type=urn:ietf:params:oauth:grant-type:jwt-bearer&assertion=eyJhbGciOiJSUzI1NiJ9.eyJleHAiOjE0NTgxNjY5ODUsInN1YiI6ImFkbWluIiwibmJmIjoxNDU4MTA2OTg1LCJhdWQiOlsiaHR0cHM6XC9cL2xvY2FsaG9zdDo5NDQzXC9vYXV0aDJcL3Rva2VuIiwid3NvMi1JUyJdLCJpc3MiOiJqd3RJRFAiLCJqdGkiOiJUb2tlbjU2NzU2IiwiaWF0IjoxNDU4MTA2OTg1fQ.ZcxdoTVEsWoil80ne42QzmsfelMWyjRZJEjUK1c2vMZJjjtrZnsWExyCA5tN6iXYFAXC_7rkFuuNSgOlBi51MNLPZw3WcgGI52j6apGEW92V2tib9zRRWOeLQLAdo8ae8KzLp7kuKZ2XunfQ2WYU9TvvLDm_vp5ruuYz3ZZrJOc' https://localhost:9443/oauth2/token 
+	```
+
+!!! tip
+    If you have configured the service provider and identity provider in a tenant, add the tenant domain as a query parameter to the access token endpoint, e.g., if the tenant domain is wso2.com, the access token endpoint should be `https://localhost:9443/oauth2/token tenantDomain=wso2.com.`
```
Comment (gomathyk) [inline en/docs/guides/access-delegation/configure-jwt-grant.md:83]: Shall we break this into two sentences?

File: en/docs/samples/saml2-bearer-assertion-profile.md [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+
+    ![request-oauth2-access-token](../assets/img/samples/request-oauth2-access-token.png)  
+
+    You will receive an access token as shown below.
+
+    ![oauth2-token-details](../assets/img/samples/oauth2-token-details.png) 
+
+6.  Use the introspection endpoint of WSO2 Identity Server to get the token information.
+
+    **Request**
```
Comment (gomathyk) [inline en/docs/samples/saml2-bearer-assertion-profile.md:90]: Shall we use tabs for these requests and responses?

Comment (sherenewso2) [reply en/docs/samples/saml2-bearer-assertion-profile.md:90]: we have used tabs to give the request format and sample request

**Pull request merged**